### PR TITLE
[CUSTCOM-88] Minor changes to DynamicInterceptor

### PR DESCRIPTION
**Pull request by fturizo**
# CUSTCOM-88 Minor changes to DynamicInterceptor

# Description
Minor internal code reformat to the `com.sun.enterprise.v3.admin.DynamicInterceptor` class. Removed un-needed `System.out.println` that was polluting the logs.

# Important Info

# Testing

### New tests
No tests needed.

### Testing Performed
Run the default test suite in the repository.

### Testing Environment
Windows 10, Zulu 11.35

### CODE REVIEW

File: nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java [hunk 1]
```diff
@@ -51,39 +52,39 @@
...
+ * since most methods won't be overridden.
  */
-public class DynamicInterceptor implements MBeanServer
-{
+public class DynamicInterceptor implements MBeanServer {
     private volatile MBeanServer mDelegateMBeanServer;
     private static final HashMap<String, MBeanServerConnection> instanceConnections =
-            new HashMap<String, MBeanServerConnection>();;
+            new HashMap<String, MBeanServerConnection>();
+    ;
```
Comment (MarkWareham) [inline nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java]: unnecessary semi colon

File: nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java [hunk 2]
```diff
@@ -40,6 +40,7 @@
 package com.sun.enterprise.v3.admin;
```
Comment (MarkWareham) [inline nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java]: Portions copyright

**Comment by MarkWareham** [comment]
jenkins test please

**Comment by MarkWareham** [comment]
sorry. Button aim was off ^

File: nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java (hunk 1 above)
Comment (fturizo) [reply nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/DynamicInterceptor.java]: Thanks, missed it. Updated.

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**